packages/paypal-commerce-utils/src/paypal-commerce-fastlane-utils.ts

```diff
@@ -7,7 +7,7 @@ import {
     PaymentMethodClientUnavailableError,
     UntrustedShippingCardVerificationType,
 } from '@bigcommerce/checkout-sdk/payment-integration-api';
-import { BrowserStorage } from '@bigcommerce/checkout-sdk/storage';
+import { CookieStorage } from '@bigcommerce/checkout-sdk/storage';
```

Comment:
I think would be better to use existed solution for handling cookies from here `packages/core/src/common/storage/browser-storage.ts`. We use it with whole project if we need to use cookies, What do you think?